scripts/k8s-galera

```diff
@@ -0,0 +1,60 @@
+#!/bin/sh
+# [This file is part of the zabbix-agent-osso package]
+# REQUIRES: cut, jq, kubectl
+#
+
+get_contexts() {
+    kubectl config get-contexts -oname
+}
+
+get_namespaces() {
+    kubectl --context "$1" get namespace \
+        -l ossobv/zabbix-agent-osso.k8s-galera=true \
+        -o go-template='{{range .items}}{{.metadata.name}}{{"\n"}}{{end}}' \
+        2>/dev/null
+}
+
+get_clusters() {
+    kubectl --context "$1" -n "$2" \
+        get mariadbs.k8s.mariadb.com -oname 2>/dev/null |
+        cut -d'/' -f2
+}
+
+make_json_clusters() {
+    echo '{"{#CONTEXT}":"'$1'","{#NAMESPACE}":"'$2'",'"\
+        "'"{#CLUSTER}":"'$3'"}'
+}
+
+make_json_cluster_status() {
+    kubectl --context "$1" -n "$2" get mariadbs.k8s.mariadb.com "$3" -o json |
+        jq -c '{
+            currentPrimaryPodIndex: .status.currentPrimaryPodIndex,
+            replicas: .status.replicas,
+            galeraReadyStatus: (.status.conditions[] | select(.reason == "GaleraReady") | .status),
```

Comment:
Sorry, 80 column limit. Please add some linefeeds.

But the code is vastly improved now, with a single kubectl call and a single jq. Good job :ok_hand: 

One question: does this fail gracefully? If kubectl cannot connect, or if jq doesn't find all elements (because let's say there is no GaleraReady reason), will Zabbix get a ZBX_NOTSUPPORTED and as a result not give us any triggers?

Reply:
I have resolved this in my latest commit